mis_builder/models/mis_report.py

```diff
@@ -601,10 +601,9 @@ def _fetch_queries(self, date_from, date_to, get_additional_query_filter=None):
                     ]
                 )
             else:
-                datetime_from = _utc_midnight(date_from, self._context.get("tz", "UTC"))
-                datetime_to = _utc_midnight(
-                    date_to, self._context.get("tz", "UTC"), add_day=1
-                )
+                tz = self._context.get("tz", "UTC") or "UTC"
```

Comment:
Do you know where Odoo implies that no timezone means UTC ?

Reply:
This https://github.com/odoo/odoo/blob/91d6598221ac50c839df749a2020846d89056daa/odoo/addons/base/models/ir_fields.py#L231 is an example

Reply:
Oh, thanks that's interesting. Maybe we could then use that `_input_tz` function here too, to be consistent with Odoo does ?

Reply:
Right, done, thanks